components/inset-text/src/fixtures.js

```diff
@@ -4,6 +4,6 @@ import InsetText from '.';
 
 export const insetText = 'It can take up to 8 weeks to register a lasting power of attorney if there are no mistakes in the application.';
 
-export default () => <InsetText>{text('children', insetText)}</InsetText>;
+export default () => <InsetText isNarrow={boolean('isNarrow', false)}>{text('children', insetText)}</InsetText>;
```

Comment:
setting isNarrow false as a prop means that the test for defaultProps isNarrow false won't be covered

Reply:
Added test and added isNarrow to propTypes! Cheers!